template/Dockerfile.debian

```
@@ -26,7 +27,20 @@ RUN set -eux; \
 	apt-get purge -y --auto-remove -o APT::AutoRemove::RecommendsImportant=false; \
 	chmod +x /usr/local/bin/gosu; \
 	gosu --version; \
-	gosu nobody true
+	gosu nobody true; \
+	echo '{{
+		{
+			name: "gosu",
+			version: .gosu.version,
+			params: {
+				os_name: "debian",
+				os_version: .debian.suite
+			},
+			licenses: [
+				"Apache-2.0"
+			]
+		} | sbom | tostring
+	}}' > /usr/local/gosu.spdx.json
```

Rather than just hoping that the quoting will always work correctly, we can just drop the outer quotes and let `jq` escape the string for shell. (We should add this change to all the `sbom` jq invocations that we have added).

```suggestion
	echo {{
		{
			name: "gosu",
			version: .gosu.version,
			params: {
				os_name: "debian",
				os_version: .debian.suite
			},
			licenses: [
				"Apache-2.0"
			]
		} | sbom | tostring | @sh
	}} > /usr/local/gosu.spdx.json
```